safety_gymnasium/utils/random_generator.py

```diff
@@ -58,7 +58,7 @@ class RandomGenerator:
 
     def __init__(self):
         """Initialize the random number generator."""
-        self.random_generator: np.random.mtrand.RandomState = None
+        self.random_generator: np.random.RandomState = None  # pylint: disable=no-member
```

Comment:
`numpy.random.RandomState` is deprecated. Open a new PR to change this to `numpy.random.Generator`. (breaking change, need a revision bump).

Reply:
Got it, thanks a lot